### PR TITLE
Marcin

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

**Comment by PietroLiuzzo** [comment]
Dear @MarcinKrawczuk and @DariaElagina could this PR be closed and commits pushed to a branch before making it a PR? thanks a lot!

**Comment by PietroLiuzzo** [comment]
no merging?

**Comment by DariaElagina** [comment]
not yet, it is still work in progress

**Comment by DariaElagina** [comment]
is that a problem?

**Comment by PietroLiuzzo** [comment]
then proceed this way until you can merge, next time, please make it a PR only when ready for review. Thanks!

**Comment by DariaElagina** [comment]
sure

File: VaticanBAV/et/BAVet46.xml [hunk 1]
```diff
@@ -42,14 +42,18 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <ptr target="bm:Lusini1996Ewostatewos"/>
                            <citedRange unit="page">354-356</citedRange>
                         </bibl>
+                        <bibl>
+                          <ptr target="bm:Taddesse1972Church"/>
+                          <citedRange unit="page">208f</citedRange>
+                        </bibl>
```
Comment (DariaElagina) [inline VaticanBAV/et/BAVet46.xml:48]: Could you please provide the whole range of pages and add a change element with the information on adding a bibliographical refernce.

File: VaticanBAV/et/BAVet126.xml [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                    </history>
+                    <additional>
+                       <adminInfo>
+                          <recordHist>
+                             <source>
+                                <listBibl type="catalogue">
+                                   <bibl>
+                                       <ptr target="bm:GrebTiss1935Codices"/>
+                                      <citedRange unit="page">503</citedRange>
+                                   </bibl>
```
Comment (DariaElagina) [inline VaticanBAV/et/BAVet126.xml:171]: I have added a bibliographical refernce here

File: VaticanBAV/et/BAVet126.xml [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+                          </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                      <origin>
+                        <origDate when="1899">
+                          The <ref target="#a2">subscription</ref> states that the manuscript was written in the year 1899 during the reign
+                          of <persName ref="PRS7006Menilek">Mәnilәk II</persName> when the metropolitan
+                          was <persName ref="PRS6876Matewos">Mātewos</persName>.
+                        </origDate>
```
Comment (DariaElagina) [inline VaticanBAV/et/BAVet126.xml:160]: I have slightly changed the encoding here, in accordance with the following example 1: https://betamasaheft.eu/Guidelines/?id=origin

**Pull request merged**